release.sh

```diff
@@ -1,15 +1,14 @@
 # Before you release
-# - install pandoc on your system: apt-get install pandoc
-# - install pypandoc: pip install pypandoc
 # - install twine: pip install twine
-# - bump package version in setup.py
+# - bump package version in setup.cfg
 # - update the package version in the CHANGES file
 # - commit the changes to master and push
 
-PKG_NAME=$(python setup.py --name)
-PKG_VERSION=$(python setup.py --version)
+PKG_NAME = pycobertura
+# from importlib import metadata is unfortunately only available from Python 3.8 onwards
+PKG_VERSION = python -c "from pkg_resources import get_distribution; print(get_distribution('pycobertura').version)"
 
-git tag -am "release v${PKG_VERSION}" v${PKG_VERSION}
+git tag -am "release v${PKG_VERSION}" "v${PKG_VERSION}"
 git push --tags
-python setup.py sdist
-twine upload dist/${PKG_NAME}-${PKG_VERSION}.tar.gz
+python -m build
+twine upload dist/"${PKG_NAME}-${PKG_VERSION}*"
```

Comment:
What is the reason for changing this to a wildcard instead of the more explicit `.tar.gz`?

Reply:
I think we should also upload the .whl file, see the following article: https://realpython.com/python-wheels/
Principally the documentation even recommends `twine upload dist/*`. 
But we could also just do the .tar.gz and the .whl in 2 lines.

Reply:
Oh, interesting... Yeah, then let's wheel it up!

Can we be more explicit about the `.whl` format? Maybe something like `twine upload dist/${PKG_NAME}-${PKG_VERSION}.{tar.gz,whl}`. The only reason to make it more explicit is to remember to consider the wheel file next time we need to modify this script.